capellambse/extensions/reqif/elements.py

```diff
@@ -298,10 +298,17 @@ def __repr__(self) -> str:
 
     def _short_repr_(self) -> str:
         mytype = type(self).__name__
+        if mytype.startswith("Enum"):
+            value = ", ".join(self.values.by_long_name)
+        elif mytype.startswith("Date"):
+            value = helpers.strf_datevalue(self.value)
+        else:
+            value = self.value
+
+        definition_name = re.sub(r"(?<!^)([A-Z])", r" \1", mytype)
         if self.definition is not None:
-            return f"<{mytype} {self.definition.long_name!r} ({self.uuid})>"
-        default_name = re.sub(r"(?<!^)([A-Z])", r" \1", mytype)
-        return f"<{mytype} [{default_name}] ({self.uuid})>"
+            definition_name = self.definition.long_name
+        return f"<{mytype} [{definition_name}] {value!r} ({self.uuid})>"
```

Comment:
Taking the `repr()` of a formatted date string here makes the output look like it was a `str`. It's not too big of an issue, because it says that it's a `Date` right before, but it could still be confusing. However, we could easily resolve it by taking the `repr()` at the appropriate places during the type checks above and dropping `!r` down here.